code/modules/modular_computers/computers/item/computer.dm

```diff
@@ -210,6 +216,13 @@
 		add_overlay("bsod")
 		add_overlay("broken")
 
+/obj/item/modular_computer/proc/update_label()
```

Comment:
Where is this proc called?

Reply:
Oh, I forgot to add the changes to the ID card slot, I'll add them later tonight.